### PR TITLE
Clarify how to get data from Line3D and fix formatting issue

### PR DESCRIPTION
**Pull request by oscargus**
## PR summary

Related to #26146 (maybe closes it).

## PR checklist

- [ ] "closes #0000" is in the body of the PR description to [link the related issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- [ ] new and changed code is [tested](https://matplotlib.org/devdocs/devel/testing.html)
- [ ] *Plotting related* features are demonstrated in an [example](https://matplotlib.org/devdocs/devel/documenting_mpl.html#writing-examples-and-tutorials)
- [ ] *New Features* and *API Changes* are noted with a [directive and release note](https://matplotlib.org/devdocs/devel/coding_guide.html#new-features-and-api-changes)
- [ ] Documentation complies with [general](https://matplotlib.org/devdocs/devel/documenting_mpl.html#writing-rest-pages) and [docstring](https://matplotlib.org/devdocs/devel/documenting_mpl.html#writing-docstrings) guidelines

### CODE REVIEW

**Comment by rcomer** [comment]
I agree there is not much else we can do here, so have linked the issue.

File: lib/mpl_toolkits/mplot3d/art3d.py [hunk 1]
```diff
@@ -183,6 +183,10 @@ def text_2d_to_3d(obj, z=0, zdir='z'):
 class Line3D(lines.Line2D):
     """
     3D line object.
+
+    .. note:: Use `get_data_3d` to obtain the data associated with the line.
+            `~.Line2D.get_xdata` and `~.Line2D.get_ydata` will return the x- and
+            y-coodinates of the projected line, not the x- and y-data of the 3D-line.
```
Comment (rcomer) [inline lib/mpl_toolkits/mplot3d/art3d.py:189]: Should this mention the setters as well, for completeness?

Comment (oscargus) [reply lib/mpl_toolkits/mplot3d/art3d.py:189]: Good idea! I'll add it.

Comment (rcomer) [reply lib/mpl_toolkits/mplot3d/art3d.py:189]:
```suggestion
            y-coordinates of the projected line, not the x- and y-data of the 3D-line.
```

Comment (oscargus) [reply lib/mpl_toolkits/mplot3d/art3d.py]: Whoops! Thanks!

**Review by rcomer — APPROVED** [review]
Thanks @oscargus this looks good to me.  The many test failures are troubling but clearly not related to this change.

**Pull request merged**